data/add_status.python

```diff
@@ -0,0 +1,79 @@
+import os
+import json
+
+# List of ongoing shortcodes
+ongoing_shortcodes = [
+		"0102",
+		"0103",
+		"0105",
+		"0106",
+		"0107",
+		"0112",
+		"0114",
+		"0116",
+		"0118",
+		"0119",
+		"0121",
+		"0801",
+		"0804",
+		"0805",
+		"0806",
+		"0807",
+		"080C",
+		"080E",
+		"0810",
+		"0812",
+		"0813",
+		"0816",
+		"081B",
+		"0820",
+		"0827",
+		"082B",
+		"082C",
+		"0836",
+		"0838",
+		"083A",
+		"083B",
+		"083C",
+		"083E",
+		"0843",
+		"0844",
+		"0849",
+		"084A",
+		"084E"
+]
+
+def update_json_files(folder_path):
+    print (ongoing_shortcodes)
```

Comment:
nitpick (this is not about python, I know): this looks a lot like Python 2 to me. In modern Python, `print` is a function like any other, so one should not include a space before parenthesis.

Reply:
=> https://github.com/dasch-swiss/dsp-meta/pull/146/commits/81ca39f7b79773952afc09b93b42a95da60aa6b3